docs/pages/teamshares/changelog.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 2.0.1
+## 2.0.2
```

Comment:
I think a PR this size warrants a bump up by minor version, to 2.1.0.

Reply:
Ah, this was me fixing my typo from the last patch release (where I repeated 2.0.1 twice). Now moot since we're releasing everything as 2.1.0 with the new upgarde!